packages/code-du-travail-frontend/pages/modeles-de-courriers/[slug].js

```diff
@@ -41,6 +57,8 @@ class ModeleCourrier extends React.Component {
     if (status === 404) {
       return <Answer emptyMessage="Modèle de document introuvable" />;
     }
+    const [, extension] = filename.split(/\.([a-z]{2,4})$/);
```

Comment:
Do you know in advance the only extensions that can does through here ?
This regex seems pretty little for a extension matcher :thinking:

Reply:
only `docx` for now.  maybe `odt` or `pdf`on day.
Not sure to see your concerns here